### PR TITLE
chore: beanstalk cleanup

### PR DESCRIPTION
**Pull request by ebezzi**
#### Reviewers
**Functional:** 
@MDunitz 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: server/eb/Makefile [hunk 1]
```diff
@@ -5,58 +5,10 @@ clean:
...
-	  cp -r customize/plugins artifact.dir/server/;  \
-	fi; \
-	(cd artifact.dir; \
-	  cp -r server/common/web/static static; \
-	  zip -r ../artifact.zip . --exclude server/tests/\* server/eb/\* ; ); \
-	if ! grep artifact.zip .elasticbeanstalk/config.yml ; then \
-	    mkdir -p .elasticbeanstalk ; cat config_deploy.yaml >> .elasticbeanstalk/config.yml ; fi
-
 .PHONY: build-ecs
 build-ecs: clean
```
Comment (MDunitz) [inline server/eb/Makefile:13]: can this be moved to the base makefile?

**Review by MDunitz — APPROVED** [review]
lgtm with one small nit

**Comment by atolopko-czi** [comment]
Should we also remove or rename the `eb` directory? (in a new branch/PR)

**Comment by atolopko-czi** [comment]
The EB AWS resource hasn't been destroyed. Is that future work?

**Pull request merged**